Source/WebCore/page/LocalDOMWindow.h

```diff
@@ -438,7 +445,19 @@ class LocalDOMWindow final
     mutable RefPtr<CloseWatcherManager> m_closeWatcherManager;
 
     // Equivalent to the list of PerformanceEventTiming objects mentioned in https://www.w3.org/TR/event-timing/#sec-modifications-HTML :
-    Vector<PerformanceEventTiming::Candidate, 6> m_performanceEventTimingCandidates;
+    Vector<PerformanceEventTimingCandidate, 6> m_performanceEventTimingCandidates;
+
+    // FIXME: support multiple pointers by replacing std::optional
+    // with map objects using PointerID as key:
+    Markable<EventTimingInteractionID> m_pointerMap;
+    std::optional<PerformanceEventTimingCandidate> m_pendingPointerDown;
+
+    bool m_contextMenuTriggered { false };
+
+    // StdMap is used instead of HashMap to avoid collisions with the empty key:
+    StdMap<int, PerformanceEventTimingCandidate> m_pendingKeyDowns;
```

Comment:
We should use just use `SignedWithZeroKeyHashTraits`. See `PointerCaptureController` for an example.